presidio-analyzer/pylintrc

```diff
@@ -10,7 +10,7 @@ reports=no
 disable=import-error,too-few-public-methods,missing-docstring,locally-disabled,fixme,cyclic-import,too-many-arguments,invalid-name,duplicate-code,bad-continuation
 
 [FORMAT]
-max-line-length=89
```

Comment:
This file is irrelevant now

Reply:
Yes, but it's under your commit. I don't want to do too much changes which are related to pylint and analyzer.

Reply:
Still, I've removed this file in my PR. 
You making changes in it now made my PR dependent in yours.
It's fine, I'll rebase again, but try to keep the PR related to the change, also with the pipeline reformatting